libcontainer/cgroups/systemd/apply_systemd.go

```diff
@@ -159,8 +160,36 @@ func UseSystemd() bool {
 			}
 		}
 
+		// Assume we have the ability to start a transient unit as a slice
+		// This was broken until systemd v229, but has been back-ported on RHEL environments >= 219
+		// For details, see: https://bugzilla.redhat.com/show_bug.cgi?id=1370299
```

Comment:
Can I just clarify here what you mean by "broken"? Do you mean that the interface didn't exist / always returned an error? Or do you mean that there was some issues with how systemd treated such slices (in which case, is the code you've added sufficient to protect against that)? I'm just asking because we've had some horrible issues with systemd's DBus interface and I generally don't trust that project to handle things sanely anymore.